catalog/spatial/registry/registry-publication-update-handler/src/main/java/org/codice/ddf/registry/publication/RegistryPublicationHandler.java

```diff
@@ -41,14 +46,25 @@
 
     private static final int SHUTDOWN_TIMEOUT_SECONDS = 60;
 
+    private static final String KARAF_LOCAL_ROLE = "karaf.local.roles";
+
     private final RegistryPublicationService registryPublicationService;
 
     private final ExecutorService executor;
 
+    private javax.security.auth.Subject subject;
+
     public RegistryPublicationHandler(RegistryPublicationService registryPublicationService,
             ExecutorService service) {
         this.registryPublicationService = registryPublicationService;
         this.executor = service;
+
+        Set<Principal> principals = new HashSet<>();
```

Comment:
Yeah just noticed it on this PR as well here: https://github.com/ani6gup/ddf/blob/3029a549e4f9c086220fe29761502595250627e2/catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/IdentityNodeInitialization.java#L74